tests/php/general/test_class.jetpack.php

```diff
@@ -840,7 +842,8 @@ function test_get_assumed_site_creation_date_only_admins() {
 			'user_registered' => '1992-01-01 00:00:00',
 		) );
 
-		$this->assertEquals( '1994-01-01 00:00:00', Jetpack::get_assumed_site_creation_date() );
+		$jetpack = new MockJetpack();
+		$this->assertEquals( '1994-01-01 00:00:00', $jetpack::connection()->get_assumed_site_creation_date() );
```

Comment:
Should we also add a test to ensure we're not doing an extra db query? I'm fine with not needing to do it, as this would kinda be testing the WP core transient functionality, but just wanted to confirm if you thought about it.

Reply:
Thought a little bit, but can't think of how it's worth it for this one.